audio/out/ao_wasapi_utils.c

```diff
@@ -945,7 +945,7 @@ bool wasapi_thread_init(struct ao *ao)
 {
     struct wasapi_state *state = ao->priv;
     MP_DBG(ao, "Init wasapi thread\n");
-    int64_t retry_wait = 1;
+    int64_t retry_wait = 1000;
```

Comment:
Not really important for this PR. But maybe we could have macros for defining wait times in more readable way

like 

```
#define MP_TIME_S(s) ((s) * 1000000000)
#define MP_TIME_MS(ms) ((ms) * 1000000)
#define MP_TIME_US(ms) ((ms) * 1000)
#define MP_TIME_NS(ns) ((ns))
```
then we can abstract all zeros and multiplications. and just do `MP_TIME_MS(1)`, especially we do a lot `MP_TIME_S(1)` conversions.

Reply:
Amusingly, `MP_SECOND_US` already exists apparently. What an awful name.